tron/api/resource.py

```diff
@@ -310,10 +311,16 @@ def get_data(
         include_action_runs=False,
         include_action_graph=True,
         include_node_pool=True,
+        page=None,
+        page_size=None,
     ):
+        jobs = self.job_collection.get_jobs()
+        if page is not None and page_size is not None:
+            start = (page - 1) * page_size
+            jobs = jobs[start : start + page_size]
```

Comment:
```suggestion 
        if page is not None and page_size is not None:
            jobs = self.job_collection.get_jobs_by_page(
                page,
                page_size,
            )
        else:
            jobs = self.job_collection.get_jobs()
```
might make testing easier (since then we can test the paging logic without testing the rest of `get_data`)